Doc/c-api/init.rst

```diff
@@ -221,6 +221,13 @@ to 1 and ``-bb`` sets :c:data:`Py_BytesWarningFlag` to 2.
    Set by the :option:`-v` option and the :envvar:`PYTHONVERBOSE` environment
    variable.
 
+.. c:var:: int Py_UTF8Mode
+
+   Enables the UTF-8 Mode.
+
+   If set to 1, enable the UTF-8 Mode.
+   If set to 0, disable the UTF-8 Mode.
```

Comment:
I'm not sure that it works this way. preconfig_get_global_vars() ignores Py_UTF8Mode if ``Py_UTF8Mode <= 0``.

Py_UTF8Mode should not be used: PyPreConfig.utf8_mode should be used instead. Today, it seems like the only purpose of Py_UTF8Mode is to explicitly enable the Python UTF-8 Mode with Py_UTF8Mode=1.